jvmquake.c

```diff
@@ -155,6 +165,57 @@ gc_finished(jvmtiEnv *jvmti) {
             err = (*jvmti)->RawMonitorExit(jvmti, lock);
             error_check(err, "Failed to notify killer thread");
         }
+    } else if (val > opt_gc_warning_threshold && opt_gc_warning_path != NULL) {
+        fprintf(stderr,
+                "(jvmquake) Above GC warning threshold [%lds]: touching (%s)\n",
+                opt_gc_warning_threshold / NANOS, opt_gc_warning_path);
+        int fd = open(opt_gc_warning_path, O_CREAT | O_WRONLY, 0666);
+        if (fd >= 0) {
+            futimes(fd, NULL);
```

Comment:
close the file here?

Reply:
Or keep it open and note its FD for next time?

Reply:
I closed it, this shouldn't happen often enough to save it for later IMO.